configuration/concepts/lab_tests.csv

```diff
@@ -7,17 +7,17 @@ a6f85472-1bf5-4a05-bb38-7769ef3c6350,,,Small,Pequeña,Pequeña,Small,,Misc,N/A,,
 3364e45c-e10d-4697-bb44-c26ff29a5e3a,,,Moderate,Moderada,Moderada,Moderate,,Misc,N/A,,,,,,,,,
 a0be77d9-3134-4877-b18a-f315f9c681f8,,,Large,Grande,Grande,Large,,Misc,N/A,,,,,,,,,
 97bc3471-a39c-44ce-8d01-f241e1418e41,,,Hematocrit,Hematocrito,Hematocrito,Hematocrit,,LabTest,Numeric,,,,Yes,%,39,52,,
-796a4dd1-a729-4d27-bea2-aa07496f303f,,,White Blood Cells,Células blancas de la sangre,Células blancas de la sangre,White Blood Cells,,LabTest,Numeric,,,,Yes,X10 *9L,5,10,,
-48c0720a-4850-4f32-9348-c54a78bdceb5,,,Platelets,Plaquetas,Plaquetas,Platelets,,LabTest,Numeric,,,,,X10 * 9L,150,400,,
-54057e88-5668-494c-94a3-afd13dc745ea,,,Red Blood Cells,Las células rojas de la sangre,Las células rojas de la sangre,Red Blood Cells,,LabTest,Numeric,,,,Yes,X10 *12L,4,5,,
+796a4dd1-a729-4d27-bea2-aa07496f303f,,,White Blood Cells,Células blancas de la sangre,Células blancas de la sangre,White Blood Cells,,LabTest,Numeric,,,,Yes,X10 ^9L,5,10,,

```

Comment:
Let's change the unit of 'X10 9L' to 'x10^9/L'
Let's do this for other tests as well

Reply:
This is done